commands/cum.js

```diff
@@ -0,0 +1,14 @@
+//penis pie makes the chicken cry
+//
+//
+
+exports.commandName = 'cum';
+exports.run = async (client,message,args) => {
+    let rand = Math.round(100*Math.random());
+    console.log(rand);
```

Comment:
Do we really need this to be logged?

Reply:
probably not, I will catch this in my next run through for the janClean branch